.github/ISSUE_TEMPLATE/libmamba-bug-report.yml

```
@@ -0,0 +1,84 @@
+name: Libmamba Bug Report (Experimental Feature)
```

```suggestion
name: Libmamba Solver Feedback (Experimental Feature)
```
Let's make sure we're not using the wrong name for this, as libmamba isn't the one we're asking for feedback but the libmamba-based solver. I would also suggest to make this a bit different in framing it a bit more generically as "feedback" than a "bug report" as that reduces the barrier a bit, e.g. not every issue users may find with the solver may be able identifiable as a bug for them. For example we also want to get other feedback like user experience problems etc.

As such, could you please also rename the template file name to `libmamba-solver-feedback.yml`?